src/services/NotesServices/addNoteService.js

```diff
@@ -0,0 +1,13 @@
+import axios from "axios"
+export const addNote = async (authToken, note, dispatchNotes) => {
+    try {
+        const { data } = await axios.post('/api/notes', { note }, {
+            headers: {
+                authorization: authToken
```

Comment:
Incorporating the token variable here can cause issues post deployment, the solution to which is to call the token here itself.

Feel free to reach out if that doesn't pan out.

Reply:
thank you for letting me know this